crates/auto_update/src/auto_update.rs

```diff
@@ -88,6 +88,34 @@ struct JsonRelease {
     url: String,
 }
 
+struct MacOSUnmounter {
```

Comment:
Can we follow [Rust naming conventions](https://rust-lang.github.io/api-guidelines/naming.html) here?

```suggestion
struct MacOsUnmounter {
```